onnxruntime/core/providers/openvino/ov_versions/data_ops.h

```diff
@@ -0,0 +1,67 @@
+#pragma once
+#include <unordered_set>
+
+namespace onnxruntime {
+namespace openvino_ep {
+
+typedef enum version_num { 
+  V_2020_4,
+  V_2021_1,
+  V_2021_2
+} version_id_e;
+
+using ConfirmationFunction = std::function<bool(const Node*, const Provider_InitializedTensorSet&)>;
+
+typedef struct supportedop {
+  version_id_e version;
+  std::vector<std::string> device_type;  
+  std::string optype;
+} supportedop_t;
+
+typedef struct unsupportedopmode {
+  std::vector<version_id_e> ver;
+  ConfirmationFunction func; 
```

Comment:
Should we also add a OpSet version that is supported for every node?

Reply:
Since we do not use OpSet version anywhere, I dont foresee a usecase where it will be needed